app/talk/latest-comment-link.cjsx

```diff
@@ -31,15 +26,29 @@ module?.exports = React.createClass
     title: false
     preview: false
 
+  getInitialState: ->
+    commentUser: null
+    latestCommentText: ''
+
   projectPrefix: ->
     if @props.project then 'project-' else ''
 
   lastPage: ->
     Math.ceil @props.discussion.comments_count / PAGE_SIZE
 
+  componentWillMount: ->
+    comment = @props.comment or @props.discussion?.latest_comment
+    return unless comment
+    apiClient.type('users').get(comment.user_id).then (commentUser) =>
+      @setState {commentUser}
+
+  componentDidMount: ->
+    latestCommentText = @refs?.markdownText?.getDOMNode()?.textContent
+    @setState({latestCommentText}) if latestCommentText
```

Comment:
Instead of manually creating a DOM node, I'm just rendering the component invisibly and pulling the `textContent` from that.